lib/html/rose-bush/index.html

```diff
@@ -28,15 +28,15 @@
 </form>
 </div>
 <div class="span6">
-<form action="{{script}}/jobs" class="form-inline">
+<form action="{{script}}/jobs" class="form-inline"/>
+<form action="{{script}}/cycles" class="form-inline"/>
```

Comment:
This change appears to to introduce an embedded form that does not have a close tag. The original code allows the form action to change using the `formaction` attribute in the `Cycles List` button.